core/rpc/src/state/state_light.rs

```diff
@@ -153,9 +153,16 @@ impl<Block, F, B, E, RA> StateBackend<B, E, Block, RA> for LightState<Block, F,
 				Ok(header) => Either::Left(fetcher.remote_read(RemoteReadRequest {
 					block: header.hash(),
 					header,
-					key: key.0,
+					keys: vec![key.0.clone()],
```

Comment:
Should we change the RPC to support fetching multiple keys at once? I guess that would be way more efficient with light client in the backend.

Reply:
Yes - I've mentioned that in description :) I was working on fixing `state_subscribeRuntimeVersion` and `state_subscribeStorage` for light client and they're working really slow without this fix. Hope it'll make these subscriptions on light slightly faster (though still ineffective imo).

I'll create another PR for RPC just because I consider this PR as a must-have and the other as an optional. I'm still not sure - if we need subscription on light client. They could be implemented by polling from client code with the same efficiency. But I believe, at least now, we still need subscriptions-on-light, because I find it very convenient to test light client using https://polkadot.js.org/apps/#/ and it doesn't work without subscriptions.